Ontology/Willow/Capability/Parameter/Setpoint/Quantity/TemperatureSetpoint/Air/Zone Air/ActiveZoneAirTemperatureSetpoint.json

```diff
@@ -0,0 +1,16 @@
+{
+  "@id": "dtmi:com:willowinc:ActiveZoneAirTemperatureSetpoint;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Active Zone Air Temperature Setpoint"
+  },
+  "description": {"en": "The desired value associated with active zone air temperature"},
```

Comment:
We need to be verbose in our desciption here too. I believe we said it's something like "the setpoint to which the control sequence is actively adhering to from either the effective cooling or effective heating based on the active mode of the equipment being either cooling or heating."

Reply:
Agreed